GUI/python source/UMSATSGUI_support.py

```diff
@@ -52,24 +54,19 @@ def set_Tk_var():
     global che48
     che48 = StringVar()
 
-    global cmd1Text,cmd2Text
+    global cmd1Text, cmd2Text
 
     cmd1Text = StringVar()
     cmd2Text = StringVar()
 
-def btnGroundControl_lclick(p1):
-    #print(date)
-    sys.stdout.flush()
 
 def init(top, gui, *args, **kwargs):
     global w, top_level, root, running
```

Comment:
With this many global variables, I think it would be easier if this UMSATS support stuff was in its own class.